zenodo/modules/fixtures/data/licenses.json

```diff
@@ -0,0 +1,1451 @@
+[
```

Comment:
They do, I am just using original Zenodo licenses dump as and then ignore the already existing licenses.